Modules/Example3/cmdapps/CaPTkExample3CmdApp.cpp

```diff
@@ -0,0 +1,110 @@
+/** \file CaPTkExample3CmdApp.cpp
+ * \brief CLI program for showcasing the example3
+ * 
+ * The example currently showcases image inversion
+ */
+
+#include <mitkCommandLineParser.h>
+#include <mitkException.h>
+#include <mitkLogMacros.h>
+#include <mitkImage.h>
+#include <mitkIOUtil.h>
+#include <mitkImageAccessByItk.h>
+
+#include "ExampleImageInverter.h"
+
+
+/** \brief command-line app for example3 module
+ *
+ * This command-line app takes an image and inverts it.
+ */
+int main(int argc, char* argv[])
+{
+  mitkCommandLineParser parser;
+
+  /*---- Set general information about the command-line app ----*/
+
+  parser.setCategory("CaPTk Cmd App Category");
+  parser.setTitle("CaPTk Example3 Cmd App");
+  parser.setContributor("CBICA");
+  parser.setDescription(
+    "This command-line app showcases the example3 module.");
+
+  // How should arguments be prefixed
+  parser.setArgumentPrefix("--", "-");
+
+  /*---- Add arguments. Unless specified otherwise, each argument is optional.
+            See mitkCommandLineParser::addArgument() for more information. ----*/
+
+  parser.addArgument(
+    "input",
+    "i",
+    mitkCommandLineParser::String,
```

Comment:
I think there is a _mitkCommandLineParser::File_ which is technically more correct.

Reply:
another PR for example4 coming.